load.go

```diff
@@ -12,13 +13,19 @@ import (
 	"sort"
 	"strings"
 
+	"github.com/go-playground/mold/modifiers"
+
 	validator "gopkg.in/validator.v2"
 
 	// Load all default adapters of the objconv package.
 	_ "github.com/segmentio/objconv/adapters"
 	"github.com/segmentio/objconv/yaml"
 )
 
+var (
+	mod = modifiers.New()
```

Comment:
Should we export this variable to allow programs to add custom modifiers?

Reply:
I think that might be a good idea, although I can’t see adding custom modifiers often that can’t be added directly to the mold package, I can definitely see adding aliases custom to the companies workflow.

I would definitely rename it to `Modifier` or something to make it more descriptive though.